src/main/java/mods/eln/Eln.java

```diff
@@ -5515,6 +5546,13 @@ void recipeHeatFurnace() {
 				Character.valueOf('i'), findItemStack("Copper Thermal Cable"),
 				Character.valueOf('I'), findItemStack("Combustion Chamber"));
 
+		addRecipe(findItemStack("Fuel Heat Furnace"),
```

Comment:
> I'd suggest adding a cauldron in the middle and swapping out one of the side iron ingots for a motor.

Good point. I will do that.

> You may or may not add code to make it actually need power; that's not such a big deal.

For simplicity I would prefer to not to, there are already the fuel, the thermic and the control connections...

> f the smallest fuel burner doesn't have the capacity to replace at least one fully-upgraded heat furnace -- then it should.

Ok, I will check that.

Thanks for the input.